File: lessons_v2/lesson02/codes/nq026_acw668.py
# NQ: AcWing 668
a, b, c, d = map(int, input().split())
start = a * 60 + b
end = c * 60 + d
if end <= start:
    end += 24 * 60
diff = end - start
print(f"O JOGO DUROU {diff // 60} HORA(S) E {diff % 60} MINUTO(S)")

File: lessons_v2/lesson02/codes/nq026_acw668.cpp
#include <cstdio>

int main()
{
    int a, b, c, d;
    scanf("%d%d%d%d", &a, &b, &c, &d);

    int start = a * 60 + b;
    int end = c * 60 + d;

    int spent_time = end - start;
    if (spent_time <= 0) spent_time += 1440;

    printf("O JOGO DUROU %d HORA(S) E %d MINUTO(S)", spent_time / 60, spent_time % 60);

    return 0;

}